### DISCUSSION TITLE
Supabase upgrade failure

### DISCUSSION THREAD

**Discussion by maxca17** [supabase/supabase#26847] [Questions]
I attempted to update my Supabase and it has been stuck for almost a full day. Any Idea why? 
<img width="1367" alt="Screenshot 2024-05-28 at 8 39 49 PM" src="https://github.com/supabase/supabase/assets/111161763/947c6a68-7e2d-4547-bd31-80ac2d21e201">

**Comment by GaryAustin1** [comment]
Hopefully you have contacted support...
All you can attempt is to hard refresh the browser, but chances are slim that will help.

**Comment by superwoou** [comment]
Same problem here. I waste 3days :(

**Reply by GaryAustin1** [reply]
Have you contacted support or asked for help anywhere?
Lots of help on Discord and here, but in the case of this issue if your database is small it should take minutes.  And only support can fix it.
